source/extension/georef/index.md

```diff
@@ -77,6 +77,12 @@ In a Georeference Annotations, these steps are stored as follows:
 | Transformation algorithm | A `transformation` property defined on the GeoJSON Feature Collection that holds the GCPs |
 {: .api-table #table-critical-data-for-georeferencing}
 
+#### 2.2.1 Handling Projections
+
+Please note that version 1.1 of this extension has no explicit provisions for indicating which projection, if any, was used during the georeferencing process described above.
```

Comment:
Just to note we don't tend to version extensions but keep track of changes in the change log at the bottom of the document.

Reply:
Oh sounds good, my misunderstanding. I'll update that wording.